src/OpenSage.Tools.BigEditor/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using OpenSage.Data.Big;
+using OpenSage.Tools.BigEditor.UI;
+using Veldrid;
+using Veldrid.StartupUtilities;
+
+namespace OpenSage.Tools.BigEditor
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Platform.Start();
+
+            const int initialWidth = 1024;
+            const int initialHeight = 768;
+
+            VeldridStartup.CreateWindowAndGraphicsDevice(
+                new WindowCreateInfo(100, 100, initialWidth, initialHeight, WindowState.Normal, "OpenSAGE Big Editor"),
+                out var window,
+                out var graphicsDevice);
+
```

Comment:
Add `graphicsDevice.SyncToVerticalBlank = true;` here to enable VSync, which fixes some glitches with input handling and reduces CPU usage by ~10x.